assets/models/system/microsoft-rad-dino/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: huggingface/microsoft-rad-dino/1733475346/mlflow_model_folder
```

Comment:
Is the container path correct?

Reply:
Yes, this is autogenerated by the pipeline